client/extensions/woocommerce/woocommerce-services/views/carrier-accounts/style.scss

```diff
@@ -8,7 +8,11 @@
 
 		.carrier-accounts__list-item-carrier-icon {
 			width: 48px;
-			padding-left: 24px;
+			padding-left: 16px;
```

Comment:
Similar to https://github.com/Automattic/woocommerce-services/blob/develop/client/extensions/woocommerce/woocommerce-services/views/packages/style.scss#L187-L215